### PR TITLE
fix multicast for windows

### PR DESCRIPTION
**Pull request by jkralik**
send multicast message over all interfaces of OS

### CODE REVIEW

File: net/connUDP.go [hunk 1]
```diff
@@ -38,8 +132,63 @@ func (c *ConnUDP) Close() error {
...
+		written := 0
+		for written < len(buffer) {
+			select {
+			case <-ctx.Done():
+				return ctx.Err()
+			default:
+			}
+
+			if err := c.packetConn.SetMulticastInterface(&iface); err != nil {
+				// log.Errorf("cannot set multicast interface %v for connection: %v", iface.Name, err)
```
Comment (cizmazia) [inline net/connUDP.go]: Remove the stale comment, or pass errors to a callback `func(error)`? Same below.

File: net/connUDP.go [hunk 2]
```diff
@@ -38,8 +132,63 @@ func (c *ConnUDP) Close() error {
...
+				break
+			}
+
+			c.packetConn.SetMulticastHopLimit(c.multicastHopLimit)
+			err := c.packetConn.SetWriteDeadline(time.Now().Add(c.heartBeat))
+			if err != nil {
+				return fmt.Errorf("cannot write multicast with context: cannot set write deadline for connection: %v", err)
```
Comment (cizmazia) [inline net/connUDP.go]: Should this error cancel the entire multicast?

Comment (jkralik) [reply net/connUDP.go]: deadline must be always set -> if not it's means that packetConn is in bad state (eg it is closed)

File: net/connUDP.go (hunk 1 above)
Comment (jkralik) [reply net/connUDP.go]: removed

**Pull request merged**